src/components/ClusterOverview/ClusterOverviewContext/ClusterOverviewContext.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+
+export const ClusterOverviewContext = React.createContext({});
+
+export const ClusterOverviewContextGenericConsumer = ({ Component, dataPath }) => (
```

Comment:
I get that for now, it is enough, but this logic will move to the cards once they need more than one property.

Reply:
sure, this is as expected.
The wrapping (`*Connected`) is separate layer with its simplicity/complexity